sormas-api/src/main/java/de/symeda/sormas/api/dashboard/CaseDataTypeDashboardCriteria.java

```diff
@@ -0,0 +1,13 @@
+package de.symeda.sormas.api.dashboard;
+
+import de.symeda.sormas.api.caze.NewCaseDateType;
+
+public class CaseDataTypeDashboardCriteria extends DashboardCriteria {
```

Comment:
I think it would make more sense to call this `SurveillanceDashboardCriteria` so that we could use it for additional properties that are only relevant for the surveillance dashboard in the future.